virttest/utils_libvirt/libvirt_audit.py

```diff
@@ -0,0 +1,38 @@
+"""
+libvirt audit related utility functions
+"""
+
+from avocado.utils import process
+from avocado.utils import service
+
+from virttest import utils_libvirtd
+
+
+def enable_audit_log(libvirtd_config):
+    """
+    Configure audit log level"
+
+    :param libvirtd_config: libvirtd config object
+    """
+    libvirtd_config.audit_level = 1
+    libvirtd_config.audit_logging = 1
```

Comment:
Is it better to set these values as needed? I think we'd better avoid hard-codes here. What do you think?

Reply:
I automated 3 audit related cases, audit_level=1 is commonly necessary conditions since you need catch and check audit log information when validating cases. And audit_level=1 can be guarded to get full log.
But I can add one addition parameter to allow audit_level to be configured in parameter